js/core/utils/dom.js

```diff
@@ -171,12 +171,17 @@ var contains = function(container, element) {
 };
 
 var getPublicElement = function($element) {
-    if(config().useJQuery) {
-        return $element;
+    if(elementStrategy) {
+        return elementStrategy($element);
     }
     return $element && $element.get(0);
 };
 
+var setElementStrategy = function(value) {
+    elementStrategy = value;
+};
+
+exports.setElementStrategy = setElementStrategy;
```

Comment:
I am not sure if this method name is good. It is not obvious what it does. Maybe we can use something like `setPublicElementWrapper`?